api/src/main/java/org/apache/gravitino/authorization/MetadataObjectChange.java

```diff
@@ -41,6 +41,19 @@ static MetadataObjectChange rename(
     return new RenameMetadataObject(metadataObject, newMetadataObject);
   }
 
+  /**
+   * Rename a metadata entity MetadataObjectChange.
+   *
+   * @param metadataObject The metadata object.
+   * @param newMetadataObject The new metadata object.
+   * @param locations The locations of the metadata object.
+   * @return return a MetadataObjectChange for the rename metadata object.
+   */
+  static MetadataObjectChange rename(
+      MetadataObject metadataObject, MetadataObject newMetadataObject, List<String> locations) {
```

Comment:
I think we can use this new function to replace `static MetadataObjectChange rename(MetadataObject metadataObject, MetadataObject newMetadataObject)` old function.
We try to keep the function interface as small and concise as possible.

Reply:
OK.

Reply:
OK, All is replaced.